### ISSUE TITLE
Add gaia integration tests to nightly test run

### ISSUE THREAD

**Issue by shaspitz** [cosmos/interchain-security#739]
# Problem

Once #735 is merged, we should add some form of the integration tests to our nightly test run, with gaia used as the provider. I'd also be happy making this part of the merge CI

# Closing criteria

Add gaia integration tests to merge or nightly test run pipeline

**Comment by smarshall-spitzbart** [comment]
@MSalopek I assigned you cause I bet you can knock this out in minutes :) thanks

**Comment by mpoke** [comment]
@MSalopek do we still want to do this?

**Comment by MSalopek** [comment]
Yes. This is still relevant but a nice to have.

**Comment by mpoke** [comment]
Closing issue for now.
